ros2cli_common_extension/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2cli_common_extensions</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="root@todo.todo">root</maintainer>
```

Comment:
you can add either @Blast545 or me as maintainers.

Reply:
I have taken your information as the maintainer, can you check it? @ivanpauno

Reply:
Yes, it's fine